app/templates/about/legal.hbs

```diff
@@ -9,29 +8,32 @@
   </p>
 
   <h2>Non-Legalese TL;DR</h2>
-
   <p>
-    Ember's strong brand and identity are an important part of the project. We're protective of it! Please be respectful and <a href="mailto:leah@emberjs.com">reach out</a> if you're unsure if your use is acceptable. Visit the {{link-to "branding page" "logos"}} to learn more about what is and isn't permitted, and to download logos.
+    Ember's strong brand and identity are an important part of the project. We're protective of it! Please be respectful and <a href="mailto:leah@emberjs.com">reach out</a> if you're unsure if your use is acceptable. Visit the <LinkTo @route="logos">branding page</LinkTo> to learn more about what is and isn't permitted, and to download logos.
   </p>
   <p>
     We often grant licenses for use of the Ember logo for non-commercial community use cases. If you think you might be one of them, let's talk. We always aim to be reasonable.
   </p>
   <p>
-    The Tomster in particular has many more restrictions and should not be used without express authorization. {{link-to "You can view the various official Tomsters" "mascots"}}, and check out {{link-to "our growing FAQ" "mascots.faq"}}.
+    The Tomster in particular has many more restrictions and should not be used without express authorization. <LinkTo @route="mascots">You can view the various official Tomsters</LinkTo>, and check out <LinkTo @route="mascots.faq">our growing FAQ</LinkTo>.
   </p>
 
   <h2>The Ember License</h2>
-
   <p>
     The most current Ember license details <a href="https://github.com/emberjs/ember.js/blob/master/LICENSE">can be found on GitHub</a>.
   </p>
 
   <h2>Documentation Images</h2>
-
-  <p>
-    The graphs, diagrams and charts in the <a href="/guides">Ember Guides</a> have been released under the Creative Commons Attribution-NonCommercial-ShareAlike (CC BY-NC-SA) license.
-  </p>
-  <img alt="CC, By, Non Commercial, Share Alike" class="pull-right" src="/images/legal/license-cc_by_nc_sa.png">
+  <div class="layout-grid">
```

Comment:
What I would have preferred here was more control over the way text wraps around and image, but I didn't see that in the style guide, so I pulled it off the right as it was in the original version.

Reply:
I think this is a reasonable approach using the grid helpers 🤔 I also think it works out quite well at lower resolutions.

May I ask why you're using `col-4` and not `col-5` so that you will fill the full width? I don't think it's a blocker for merging this but I'm just curious why you made that choice

Reply:
I wasn't actually sure what each grid size did. I played with it a bit on different device widths and this seemed like the best choice. Is there a way to indicate that the image should take up one unit to the right and then text should fill all remaining space? `col-fr4`, `col-fr1` for example.

Reply:
interesting... it seems like we don't document the difference between `col-3` an `col-3-large` 🙈 I'll create an issue now to fix that  documentation

right now we have a 6 column system so if you want it the text to "fill all the remaining space" then you would make it use `col-5`

we then fall back to a 4 column system on smaller screens so if you want it to still have one column for the image and the "rest" for the text then you can do `col-5-large col-3` which just means "use 5 columns on larger screens but 3 columns on smaller screens 👍

Reply:
Got it! Thanks. I added that here.

Reply:
Perfect that looks great! 🎉